oaknational/Oak-Web-Application
# feat: about leadership page

## Description

- Adds leadership about page

## Issue(s)

Fixes #405 

## How to test

1. Go to about => leadership
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
